code/modules/client/login_processing/10-load_preferences.dm

```diff
@@ -19,7 +19,8 @@
 		atklog,
 		fuid,
 		parallax,
-		2fa_status
+		2fa_status,
+		ghost_darkness_level
```

Comment:
This will merge conflict your other PR

Reply:
I am aware

Reply:
I'm not going to make assumptions however, it's easy enough to tweak this or that to reflect if either are merged.